lessons/ro/chapter_1.yaml

```
@@ -0,0 +1,227 @@
+- title: Capitolul 1 - The Basics
+  content_markdown: >
+    În acest prim capitol vom explora ideile și conceptele de bază despre
+    funcții, variabile și tipurile primitive de date. Ne bucurăm că
+    ne sunteți alături!
+
+
+    De asemenea, în cazul în care vă întrebați cine este crabul adorabil care
+    vă vorbește, mă numesc **Ferris**,
+    mascota neoficială a limbajului Rust. Încântat de cunoștință!
+
+
+    Odată ce vă veți familiariza cu Rust, puteți să vă declarați un **Rustaceu**. Așa
+    se numesc toți oamenii care folosesc, contribuie sau sunt interesați de Rust.
+- title: The Rust Playground
+  code: >-
+    https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&code=fn%20main()%20%7B%0A%20%20%20%20println!(%22Bun%20venit%20în%20locul%20de%20joacă!%20Puteți%20modifica%20codul%20aici.%22)%3B%0A%7D%0A
+  content_markdown: >
+    În acest curs veți folosi un utilitar interactiv pentru scrierea codului [Rust
+    Playground](https://play.rust-lang.org).
+
+
+    E o modalitate bună de a vă juca cu Rust și a arăta altora 
+    creativitatea și provocările dumneavoastră!
+- title: Variabile
+  code: >-
+    https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&code=fn%20main()%20%7B%0A%20%20%20%20%2F%2F%20rust%20infers%20the%20type%20of%20x%0A%20%20%20%20let%20x%20%3D%2013%3B%0A%20%20%20%20println!(%22%7B%7D%22%2C%20x)%3B%0A%0A%20%20%20%20%2F%2F%20rust%20can%20also%20be%20explicit%20about%20the%20type%0A%20%20%20%20let%20x%3A%20f64%20%3D%203.14159%3B%0A%20%20%20%20println!(%22%7B%7D%22%2C%20x)%3B%0A%0A%20%20%20%20%2F%2F%20rust%20can%20also%20declare%20and%20initialize%20later%2C%20but%20this%20is%20rarely%20done%0A%20%20%20%20let%20x%3B%0A%20%20%20%20x%20%3D%200%3B%0A%20%20%20%20println!(%22%7B%7D%22%2C%20x)%3B%0A%7D%0A
+  content_markdown: >
+    Variabilele sunt declarate folosind cuvântul cheie **let**.
+
+
+    Atunci când dai unei variabile o valoare, Rust este capabil să deducă tipul acesteia
+    în 99%
+
+    din cazuri. Dacă acesta nu poate, puteți adăuga tipul de date la declararea variabilei.
+
+
+    Observați cum putem să dăm valori aceluiași nume de variabilă de mai multe ori. Acest
+    fenomen se numește variable
+
+    shadowing (punere în umbră) și tipul de date se poate schimba pentru referințe ulterioare
+    la acel nume de variabilă.
+
+
+    Numele de variabile sunt mereu scrise sub formă `snake_case` (ex: my_first_variable).
```

```suggestion
    Numele de variabile sunt mereu scrise sub formă `snake_case` (ex: my\_first\_variable).
```
Underscores are parsed as markdown syntax (see [here](https://tourofrust.com/03_ro.html)), so you need to escape them.